pkg/apis/resources/v1alpha1/types.go

```diff
@@ -117,6 +117,9 @@ const (
 	// HighAvailabilityConfigZones is a constant for an annotation on a Namespace which describes the availability
 	// zones are used.
 	HighAvailabilityConfigZones = "high-availability-config.resources.gardener.cloud/zones"
+	// HighAvailabilityConfigZonePinning is a constant for an annotation on a Namespace which enables pinning of
+	// workload to the specified zones.
+	HighAvailabilityConfigZonePinning = "high-availability-config.resources.gardener.cloud/zone-pinning"
```

Comment:
Please adapt the docs:
- https://github.com/gardener/gardener/blob/master/docs/concepts/resource-manager.md#high-availability-config
- https://github.com/gardener/gardener/blob/master/docs/development/high-availability.md (if necessary)